include/queue/queue_base.hpp

```diff
@@ -50,6 +50,75 @@ class Queue_Interface {
    */
   template <typename T>
   inline void deallocate(T *p) const;
+
+  // This function returns Work-group size which is equal to  maximum device
+  // workgroup size.
+  inline size_t get_work_group_size() const;
+};  // namespace blastemplate<classExecutionPolicy>classQueue_Interface
+
+template <>
+class Queue_Interface<Sequential> {
+  Queue_Interface() {}
+  /*
+  @brief This class is to determine whether or not the underlying device has
```

Comment:
This class? (same for other member functions).

Reply:
Fixed